src/popover.coffee

```diff
@@ -51,6 +51,10 @@ Ember.Widgets.BodyEventListener,
     @set 'visibility', 'visible'
     @set 'isShowing', yes
 
+  willDestroyElement: ->
+    @$().off $.support.transition.end
+    @_super()
```

Comment:
Is there an important distinction between calling the super destructor first or last? I'm wondering because I notice it's last here but first in `_removeDocumentHandlers` above.